corehq/motech/repeaters/repeater_generators.py

```diff
@@ -470,11 +474,24 @@ def _check_index_relationship(self, attribute, value):
             raise DataRegistryCaseUpdateError("Index relationships must be either 'child' or 'extension'")
 
     def get_case_block(self, target_case):
+        kwargs = {}
+        if target_case is None:
+            if not self.create_case:
+                # should never get here but added as a precaution
```

Comment:
Why not?  Couldn't a misconfigured intent case trigger this?  Is it caught somewhere else earlier?

Reply:
Nevermind - I see it should be caught when fetching the target case and raise "Target case not found"